app/views/users/show.html.haml

```diff
@@ -0,0 +1,37 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(@user.email.downcase)}?s=40")
```

Comment:
This would be a great place to use a helper method for `gravatar_url` passing in user object as an argument. You will be using this in other views too. Also I would use `= image_tag ...` rather than HTML.